kie-server-parent/kie-server-api/src/test/java/org/kie/server/api/marshalling/objects/DateObject.java

```diff
@@ -0,0 +1,62 @@
+package org.kie.server.api.marshalling.objects;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlRootElement;
+import javax.xml.bind.annotation.adapters.XmlJavaTypeAdapter;
+
+import com.thoughtworks.xstream.annotations.XStreamAlias;
+import com.thoughtworks.xstream.annotations.XStreamConverter;
+import org.kie.internal.jaxb.LocalDateTimeXmlAdapter;
```

Comment:
Had they existed, I wouldn't have to come up with the PR :-) I did some research, however couldn't find any.